Dockerfile

```
@@ -0,0 +1,14 @@
+FROM rust:alpine as builder
+RUN apk add clang musl-dev
+
+COPY . /tmp
+WORKDIR /tmp
+
+RUN cargo --version
+RUN cargo build --release --target x86_64-unknown-linux-musl
```

the target can be removed, as it will only blow up the binary size. Having alpine available static linking is not needed any more.

```suggestion
RUN cargo build --release
```